mirrord/cli/src/config.rs

```diff
@@ -103,6 +103,10 @@ pub(super) struct ExecArgs {
     #[arg(short = 'l', long)]
     pub agent_log_level: Option<String>,
 
+    /// Controls whether the agent produce logs in readable tracing mode or json.
+    #[arg(long)]
```

Comment:
We don't add new flags to the cli unless those are crucial.

Reply:
Removed!